webapp/src/DOMJudgeBundle/Resources/views/jury/jury_macros.twig

```diff
@@ -1,5 +1,7 @@
 {% macro table_extrahead() %}
+    <link rel="stylesheet" href="{{ asset('css/bootstrap-toggle.min.css') }}" type="text/css"/>
```

Comment:
I think the original idea of @ubergeek42 was to put here the extra CSS/JS we need for tables (hence `table_extrahead`). You might want to define a `toggle_extrahead` or something.

Reply:
Done.